rasa/core/interpreter.py

```diff
@@ -37,20 +38,17 @@ def create(
 
         if not isinstance(obj, str):
             if obj is not None:
-                logger.warning(
+                warnings.warn(
                     "Tried to create NLU interpreter "
-                    "from '{}', which is not possible."
+                    f"from '{obj}', which is not possible."
                     "Using RegexInterpreter instead."
-                    "".format(obj)
                 )
             return RegexInterpreter()
 
         if endpoint is None:
             if not os.path.exists(obj):
-                logger.warning(
-                    "No local NLU model '{}' found. Using RegexInterpreter instead.".format(
-                        obj
-                    )
+                warnings.warn(
+                    f"No local NLU model '{obj}' found. Using RegexInterpreter instead."
```

Comment:
If you have a core only model, that's actually fine. So I'd keep the `logger.warning` one

Reply:
changed it